skeleton/build.gradle

```diff
@@ -0,0 +1,30 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-kapt'
+apply plugin: 'com.squareup.anvil'
+
+dependencies {
+    implementation deps.androidx_fragment
+    implementation deps.appcompat
+    implementation deps.constraint_layout
+    implementation deps.javax_inject
+    implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-android:1.3.9'
+    implementation deps.coroutines
+    implementation "com.dropbox.mobile.store:store4:4.0.2-KT15"
```

Comment:
Are actually using Store somewhere in the demo app? If we are great. But if we're not lets remove the reference.